packages/fxa-auth-server/config/index.ts

```diff
@@ -2181,7 +2181,7 @@ const convictConf = convict({
         format: Array,
       },
       maxMessageLength: {
-        default: 60,
+        default: 160,
```

Comment:
@dschom I saw that 160 was the limit for a deprecated Twilio endpoint, other active endpoints seem to have a limit of 1600 (which is way too much IMO for our needs). Considering that the localized body text in English is roughly 60, setting the limit here to 160 seems fair to account for expansion in other locales.

Is there a reason for the limit being set to 60 here vs 160 in `sms.manager.spec.ts`?

Reply:
160 should be fine.

Reply:
I've shortened the English message to be under 60 characters, but languages that use a different encoding like Russian and Japanese might require a higher number of characters. Hopefully comfortably under the 160 character limit.

Reply:
I read up a bit more on the encoding for SMS, and it's trickier to calculate than I thought. For now I've reverted to full length message from Figma which is about 100 characters long - this is fine for GSM encoding - and added a comment in the FTL string to check the messaging segment calculator when localizing. I've also filed a follow-up (FXA-11072) to review handling for formatted messages that exceed the max length set here.